package.json

```diff
@@ -34,27 +34,27 @@
     "devDependencies": {
         "@commitlint/cli": "17.8.1",
         "@commitlint/config-conventional": "17.8.1",
-        "@types/jest": "29.5.6",
-        "@typescript-eslint/eslint-plugin": "6.9.0",
-        "@typescript-eslint/parser": "6.9.0",
+        "@types/jest": "29.5.11",
+        "@typescript-eslint/eslint-plugin": "6.15.0",
+        "@typescript-eslint/parser": "6.15.0",
         "abortcontroller-polyfill": "1.7.5",
         "cz-conventional-changelog": "3.3.0",
-        "eslint": "8.52.0",
-        "eslint-config-prettier": "9.0.0",
+        "eslint": "8.56.0",
+        "eslint-config-prettier": "9.1.0",
         "eslint-config-typescript": "3.0.0",
-        "eslint-plugin-jest": "27.4.3",
-        "eslint-plugin-prettier": "5.0.1",
+        "eslint-plugin-jest": "27.6.0",
+        "eslint-plugin-prettier": "5.1.0",
         "husky": "8.0.3",
         "jest": "29.7.0",
         "jest-fetch-mock": "3.0.3",
         "jest-runner-eslint": "2.1.2",
-        "lint-staged": "15.0.2",
-        "prettier": "3.0.3",
-        "semantic-release": "22.0.5",
+        "lint-staged": "15.2.0",
+        "prettier": "3.1.1",
+        "semantic-release": "22.0.12",
         "ts-jest": "29.1.1",
         "tsjs": "5.0.1",
         "tslib": "2.6.2",
-        "typescript": "5.2.2"
+        "typescript": "5.3.3"
```

Comment:
I always wonder when updating typescript by a minor is it is considered breaking. Since it could introduce new features in a minor, aka new types that wouldn't be understood by previous version of TS. I quickly skimmed through the changelog and it seems not breaking in this case.